src/model/operation/operation.js

```diff
@@ -112,7 +112,7 @@ export default class Operation {
 	 * @type {String}
 	 */
 	static get className() {
-		return 'engine.model.operation.Operation';
+		return 'Operation';
 	}
```

Comment:
I think you should never ask for this property of the base class. I think we should remove this method or replace it with `throw`.

Reply:
Agree that this method should be never used (in source code) for the base class. But If you want to drop this method for `Operation` class let's do it in a separate ticket because it needs some fixes in `toJSON()` method and in tests.